scripts/gha/get_pr_review_comments.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fetches and formats review comments from a GitHub Pull Request."""
+
+import argparse
+import os
+import sys
+import firebase_github # Assumes firebase_github.py is in the same directory or python path
+
+# Attempt to configure logging for firebase_github if absl is available
+try:
+    from absl import logging as absl_logging
+    # Set verbosity for absl logging if you want to see logs from firebase_github
+    # absl_logging.set_verbosity(absl_logging.INFO)
+except ImportError:
+    pass # firebase_github.py uses absl.logging.info, so this won't redirect.
+
+def main():
+    default_owner = firebase_github.OWNER
+    default_repo = firebase_github.REPO
+
+    parser = argparse.ArgumentParser(
+        description="Fetch review comments from a GitHub PR and format for use with Jules.",
```

Comment:
Let's replace "format for use with Jules" with "format into a simple text file" so it's more generic.